opam missing 'unzip' runtime dependency

### Steps to reproduce
- `nix-env -i opam`
- `opam init`

```
[ERROR] Missing dependencies -- the following commands are required for OPAM to operate:
          - unzip
```

It doesn't seem possible to do anything useful with opam without unzip installed.
## Technical details
- System: 16.03.498.f8a5d1e (Emu)
- Nix version: nix-env (Nix) 1.11.2
- Nixpkgs version:  "16.03.498.f8a5d1e"

--- a/pkgs/development/tools/ocaml/opam/default.nix
+++ b/pkgs/development/tools/ocaml/opam/default.nix
@@ -47,7 +47,7 @@ in stdenv.mkDerivation rec {
   name = "opam-${version}";
   version = "1.2.2";
 
-  buildInputs = [ unzip curl ncurses ocaml makeWrapper];
+  buildInputs = [ unzip curl ncurses ocaml makeWrapper ];
 
   src = srcs.opam;
 
@@ -73,7 +73,7 @@ in stdenv.mkDerivation rec {
 
   postInstall = ''
     wrapProgram $out/bin/opam \
-      --suffix PATH : ${aspcud}/bin
+      --suffix PATH : ${aspcud}/bin:${unzip}/bin:${curl}/bin
   '';
 
   doCheck = false;

